### ISSUE TITLE
make lxc-master failed: PR_SET_NO_NEW_PRIVS undeclared

### ISSUE THREAD

**Issue by AlexIvamoto** [lxc/lxc#1887]
# Required information

uname -r
2.6.32-431.el6.x86_64

# Issue description

make failed:
start.c: In function ‘do_start’:
start.c:956: error: ‘PR_SET_NO_NEW_PRIVS’ undeclared (first use in this function)

# Steps to reproduce

 1. Step one
./autogen.sh - ok
 2. Step two
./configure - ok
 3. Step three
make - not ok

make
...
start.c: In function ‘do_start’:
start.c:956: error: ‘PR_SET_NO_NEW_PRIVS’ undeclared (first use in this function)
start.c:956: error: (Each undeclared identifier is reported only once
start.c:956: error: for each function it appears in.)
make[3]: *** [liblxc_la-start.lo] Error 1
make[3]: Leaving directory `/tmp/lxc-master/src/lxc'
make[2]: *** [all-recursive] Error 1
make[2]: Leaving directory `/tmp/lxc-master/src'
make[1]: *** [all] Error 2
make[1]: Leaving directory `/tmp/lxc-master/src'
make: *** [all-recursive] Error 1

**Comment by marcosps** [comment]
@AlexIvamoto 

can you please test again with a new git clone? It seems that up to 15 days ago this was fixed:

commit 995accccbb07d59381aa60939cd44b41dc092dda
Author: Peter Korsgaard <peter@korsgaard.com>
Date:   Mon Nov 6 09:35:48 2017 +0100

    lxc/start.c: Fix legacy PR_{G,S}ET_NO_NEW_PRIVS handling
    
    The configure checks for these use AC_CHECK_DECLS, which define the symbol
    to 0 if not available - So adjust the code to match.  From the autoconf
    manual:
    
    https://www.gnu.org/software/autoconf/manual/autoconf-2.65/html_node/Generic-Declarations.html)
    
    For each of the symbols (comma-separated list), define HAVE_DECL_symbol (in
    all capitals) to ‘1’ if symbol is declared, otherwise to ‘0’.
    
    Signed-off-by: Peter Korsgaard <peter@korsgaard.com>

Let me know if you face any other issues!

**Comment by brauner** [comment]
Right, that should be fixed actually.

**Comment by AlexIvamoto** [comment]
Ok, fixed, but new issue when make:

cgroups/cgfsng.c: In function ‘cgfsng_mount’:
cgroups/cgfsng.c:1687: error: ‘CAP_SYS_ADMIN’ undeclared (first use in this function)
cgroups/cgfsng.c:1687: error: (Each undeclared identifier is reported only once
cgroups/cgfsng.c:1687: error: for each function it appears in.)

will create new issue
